comms/dht/src/outbound/requester.rs

```diff
@@ -52,6 +52,33 @@ impl OutboundMessageRequester {
 
     /// Send directly to a peer. If the peer does not exist in the peer list, a discovery will be initiated.
     pub async fn send_direct<T>(
```

Comment:
```suggestion
    pub async fn send_direct_encrypted<T>(
```
We are changing the behaviour of an already existing pub API, I would force an error here to force users to look which one to use.

Reply:
Like have three functions:
- `send_direct_encrypted`
- `send_driect_unencrypted`

And then `send_direct` would now return an error?